exercises/nucleotide-count/nucleotide_count_test.go

```diff
@@ -20,6 +22,22 @@ func (h Histogram) sameMappings(o Histogram) (res bool) {
 	return
 }
 
+func assertError(count, expected int, err error, t *testing.T) {
+	var _ error = err
+	if err == nil {
+		t.Fatalf("Got \"%v\", expected \"%v\". error is nil", count, expected)
+	}
+}
+
+func assertResult(count, expected int, err error, t *testing.T) {
+	if err != nil {
+		t.Fatalf("Raised error %v when expecting none", err)
+	}
+	if count != expected {
+		t.Fatalf("Got \"%v\", expected \"%v\".", count, expected)
+	}
```

Comment:
This is only used twice, and again it's hard to figure out from reading the function call what the arguments mean. I'd prefer to inline it.